pkg/cmd/releasenotes/options.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2021 ScyllaDB
+
+package releasenotes
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"sort"
+
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	githubql "github.com/shurcooL/githubv4"
+	"golang.org/x/oauth2"
+)
+
+type GenerateOptions struct {
+	genericclioptions.IOStreams
+
+	OrganizationName string
+	RepositoryName   string
+	RepositoryPath   string
+
+	GithubToken string
+
+	ReleaseName string
+	StartRef    string
+	EndRef      string
+
+	ghClient *githubql.Client
+}
+
+func NewGitGenerateOptions(streams genericclioptions.IOStreams) *GenerateOptions {
+	return &GenerateOptions{
+		IOStreams:        streams,
+		OrganizationName: "scylladb",
+		RepositoryName:   "scylla-operator",
+		RepositoryPath:   ".",
+	}
+}
+
+func (o GenerateOptions) Validate() error {
+	if o.ReleaseName == "" {
```

Comment:
also verify release name format - so it doesn't start with `v` as you are using it for the docker image ref